### PR TITLE
adding integration framework

### PR DESCRIPTION
**Pull request by eesha-deepak**

### CODE REVIEW

**Comment by adibrastegarnia** [comment]
retest this please

**Comment by adibrastegarnia** [comment]
retest this please

File: test/p4rt/suite.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	"github.com/onosproject/helmit/pkg/input"
+	"github.com/onosproject/helmit/pkg/test"
+	"github.com/onosproject/wcmp-app/test/utils/charts"
+)
+
+type testSuite struct {
+	test.Suite
+}
+
+// TestSuite is the onos-config GNMI test suite
```
Comment (adibrastegarnia) [inline test/p4rt/suite.go:17]: Update comment no gnmi or onos-config in the comments

File: test/p4rt/suite.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+type TestSuite struct {
+	testSuite
+}
+
+func getInt(value interface{}) int64 {
+	if i, ok := value.(int); ok {
+		return int64(i)
```
Comment (adibrastegarnia) [inline test/p4rt/suite.go:24]: we can remove getInt function since there is no use of it now.

**Review by adibrastegarnia — CHANGES_REQUESTED** [review]

File: test/p4rt/suite.go [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	} else if i, ok := value.(float64); ok {
+		return int64(i)
+	} else if i, ok := value.(int64); ok {
+		return i
+	}
+	return 0
+}
+
+// SetupTestSuite sets up the onos-config GNMI test suite
```
Comment (adibrastegarnia) [inline test/p4rt/suite.go:33]: Same thing about comment here

File: test/p4rt/testP4RTConnectivity.go [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+// SPDX-License-Identifier: Apache-2.0
+
+package config
+
+import (
+	"testing"
+	"github.com/onosproject/wcmp-app/test/utils/p4rtsimulator"
+)
+
+func (s *TestSuite) testSinglePath(t *testing.T) {
```
Comment (adibrastegarnia) [inline test/p4rt/testP4RTConnectivity.go:12]: Rename the test to TestP4RTConnectivity  and we add logic later

**Comment by adibrastegarnia** [comment]
retest this please

**Comment by ray-milkey** [comment]
retest this please

**Comment by adibrastegarnia** [comment]
retest this please

**Comment by adibrastegarnia** [comment]
retest this please

**Comment by adibrastegarnia** [comment]
retest this please

**Comment by eesha-deepak** [comment]
retest this please

**Review by adibrastegarnia — APPROVED** [review]

**Pull request merged**